src/components/LocationPage/LocationPageHeading.tsx

```diff
@@ -10,10 +11,21 @@ const LocationPageHeading: React.FC<{ region: Region; isEmbed: boolean }> = ({
   } else if (region instanceof County) {
     return (
       <Fragment>
-        <strong>{region.name}, </strong>
-        <a href={`${isEmbed ? '/embed' : ''}/us/${region.state.urlSegment}`}>
-          {region.state.stateCode}
-        </a>
+        <strong>{region.name}</strong>
+        {', '}
+        <HeaderStateCode>
+          <a href={`${isEmbed ? '/embed' : ''}/us/${region.state.urlSegment}`}>
```

Comment:
I realize you didn't change this, but I think this href logic is more clear as a variable above rather than here